vision/module/get_module_depth.py

```diff
@@ -18,8 +18,13 @@ def get_module_depth(depth_image, coordinates):
     Parameters
     ----------
     depth_image: numpy array
+        The depth image.
     coordinates: tuple of numbers
-        (x, y) coordinates of the center of the module in the frame
+        (x, y) coordinates of the center of the module in the frame.
+
+    Returns
+    -------
+    float: depth at the center
```

Comment:
You might also want to add that it's in millimetres, in terms of actual depth, since that information could be important for using the function correctly.